lib/metadataTypes/MobileKeyword.js

```diff
@@ -4,7 +4,6 @@ const TYPE = require('../../types/mcdev.d');
 const MetadataType = require('./MetadataType');
 const Util = require('../util/util');
 const File = require('../util/file');
-const beautifier = require('beauty-amp-core');
```

Comment:
you moved to this util if i read correctly?

Reply:
yes. when we (I?) first did that, for some reason it wasn't included in the generic file.js lib but here in MobileKeyword.js - now it is